group.py

```diff
@@ -1,5 +1,14 @@
 """An example of how to represent a group of acquaintances in Python."""
 
-# Your code to go here...
 
-my_group =
+def main():
```

Comment:
Would be tempted to rename the function or else add "if __name__ == "__main__":" to follow Python convention. (https://stackoverflow.com/questions/4041238/why-use-def-main)